start-goss-servers.sh

```diff
@@ -74,12 +79,21 @@ goss_suites_endpoints_ports | while read suite endpoint port ; do
         continue
     fi
 
-    # Start Goss server for this entry.
-    echo "starting ${endpoint} in background on port ${port}"
-    /usr/bin/goss -g "${suite}" --vars "${tmpvars}" serve \
-        --format json --max-concurrent 4 \
-        --endpoint "/${endpoint}" \
-        --listen-addr "${ip}:${port}" &
+    if is_vshasta_node; then 
+      # Use junit for vshasta
```

Comment:
This will mean that the other scripts that call the endpoints won't work with vshasta, since they expect it to be in json format. Not sure if that is of concern.

Reply:
I do not think so... as it is now, our deployment is the only thing that runs the tests.